features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql

```diff
@@ -1877,6 +1877,55 @@ CREATE TABLE IDN_OAUTH_PAR (
 )
 /
 
+CREATE TABLE IDN_ORG_USER_INVITATION (
+	ID INTEGER NOT NULL,
+	INVITATION_ID VARCHAR(40) NOT NULL,
+	CONFIRMATION_CODE VARCHAR(40) NOT NULL,
+	USER_NAME VARCHAR(255) NOT NULL,
+	DOMAIN_NAME VARCHAR(255) NOT NULL,
+	EMAIL VARCHAR(255) NOT NULL,
+	USER_ORG_ID VARCHAR(255) NOT NULL,
+	INVITED_ORG_ID VARCHAR(255) NOT NULL,
+	USER_REDIRECT_URL VARCHAR(1024) NOT NULL,
+	STATUS VARCHAR(10) NOT NULL,
+	CREATED_AT TIMESTAMP NOT NULL,
+	EXPIRED_AT TIMESTAMP NOT NULL,
```

Comment:
Do we need to store the expiry time? can we derive based on `CREATED_AT` ? this way we can change the expiry policy based on the latest config for expiry time which can be considered both pros and cons based on the use case. Could you check similar implementation in the governance for confirmation code?

Reply:
In the device flow and session store we are keeping a seperate column to keep the expiry time.